build/panolens.js

```diff
@@ -71,19 +48,19 @@ var TWEEN = TWEEN || (function () {
 
 		},
 
-		update: function (time) {
+		update: function (time, preserve) {
```

Comment:
Looks like tween.js has been updated to 16.3.5. Can you also specify the updated version in `package.json`?